tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts

```diff
@@ -0,0 +1,684 @@
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.content.deleteAllContent();
+
+    await createContentWithOneBlockListEditor(umbracoApi, documentName, documentAlias, elementName, elementAlias, blockListName, blockListAlias);
+
+    await umbracoUi.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.refreshContentTree();
+
+    // Opens the content with the block list editor
+    await page.locator('[data-element="tree-item-' + blockListName + '"]').click();
+
+    // Deletes the block list editor inside of the content
+    await page.locator('[title="Delete"]').click();
+    await page.locator('[label-key="actions_delete"]').click();
+
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.saveAndPublish));
+
+    // Assert
+    await umbracoUi.isSuccessNotificationVisible();
+    // Checks if the content is actually deleted
+    await expect(page.locator('[ui-sortable="vm.sortableOptions"]').nth(0)).not.toBeVisible();
+
+    // Clean
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.content.deleteAllContent();
+  });
+
+  test('can copy block list content and paste it', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
+    await umbracoApi.content.deleteAllContent();
+
+    await createContentWithOneBlockListEditor(umbracoApi, documentName, documentAlias, elementName, elementAlias, blockListName, blockListAlias);
+
+    await umbracoUi.goToSection(ConstantHelper.sections.content);
+    await umbracoUi.refreshContentTree();
+
+    // Opens the content with the block list editor
+    await page.locator('[data-element="tree-item-' + blockListName + '"]').click();
+
+    // Checks to make sure that there is only one item
+    await expect(page.locator('.umb-block-list__block--view')).toHaveCount(1);
+
+    // Copies block list content
+    await page.locator('[title="Copy"]').click();
+    await expect(page.locator('.alert-success', {hasText: 'Copied to clipboard'})).toBeVisible();
+    // Pastes block list content
+    await page.locator('[title="Clipboard"]').click();
+    await page.locator('umb-block-card', {hasText: elementName}).click();
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.saveAndPublish));
+
+    // Assert
+    await expect(page.locator('.umb-block-list__block--view')).toHaveCount(2);
+    await page.locator('.umb-block-list__block--view').nth(1).click();
```

Comment:
I really do not like `-nth()`, is there maybe another way we can assert this? 😁

Reply:
I could not find another way 😞